docs/src/integrate/integration-tutorial.md

```diff
@@ -0,0 +1,252 @@
+---
+title: Integrate with the Node.js API Tutorial
+eleventyNavigation:
+    key: integrate with the node.js api tutorial
+    parent: integrate eslint
+    title: Integrate with the Node.js API Tutorial
+    order: 1
+---
+
+This guide walks you through integrating the `ESLint` class to lint files and retrieve results, which can be useful for creating integrations with other projects.
+
+## Why Create a Custom Integration?
+
+You might want to create an ESLint integration if you're creating developer tooling, such as the following:
+
+- **Code editors and IDEs**: Integrating ESLint with code editors and IDEs can provide real-time feedback on code quality and automatically highlight potential issues as you type. Many editors already have ESLint plugins available, but you may need to create a custom integration if the existing plugins do not meet your specific requirements.
+
+- **Continuous Integration (CI) pipelines**: Including ESLint in CI pipelines can help enforce code quality standards before merging changes into the main codebase. By integrating ESLint into your CI process, you can automatically run linting checks on pull requests and provide feedback to developers about potential issues.
```

Comment:
I'd need to double-check, but I think most CI systems use the CLI.

Reply:
ok, i can remove this bullet point, then